src/index.ts

```diff
@@ -184,8 +183,11 @@ export class GraphQLServer {
         return this.options.formatResponse
       }
       return (response, ...args) => {
+        const truthyValues = ['', '1', 'true']
```

Comment:
`?deduplicate` and `?deduplicate=` are the same (`''`), I think we can treat both as `true` or only allow `?deduplicate=true` and `?deduplicate=1`, otherwise we'll need to do another check

Reply:
Basically we can do something like this:
```ts
function isTrulyQueryParam(value: string) {
  return value === '1' || value === 'true'
}
```

Reply:
I'd go for treating `?deduplicate`, `?deduplicate=`, `?deduplicate=true` and `?deduplicate=1` as `true` then. I guess people would want to use `http://example.com/graphql?deduplicate` – it looks pretty clean.

Reply:
```js
/**
 * ?value ?value= ?value=1 ?value=true
 */
function isTrulyQueryParam(value: string) {
  return value === '' || value === '1' || value === 'true'
}
```